cmd/frontend/internal/frontend/service.go

```diff
@@ -293,25 +335,25 @@ func (fes *FrontEndService) Monitor(ctx context.Context, errCh chan<- error) {
 				delay = 1 * time.Second
 			}
 
+			// These errors may be temporary, so don't denounce immediately
 			protocolOut, err := fes.routingService.ShowProtocolSessions(ctx, lp, `NBR-*`)
 			if err != nil {
 				logger.Error(err, "protocol output", "out", strings.Split(protocolOut, "\n"))
-				denounce = true
+				sessionErrors++
 				continue
 			}
-
 			if strings.Contains(protocolOut, bird.NoProtocolsLog) {
 				logger.Info("protocol output", "out", protocolOut)
-				denounce = true
+				sessionErrors++
```

Comment:
This is not a "session error". It indicates that there are no gateways configured. Either not added yet (in which case it doesn't matter if it was a session error or not), or all the gateways had been removed from the Attractor (FE config).
A denounce is required.